core/src/main/scala/org/http4s/MessageFailure.scala

```diff
@@ -56,6 +56,7 @@ object ParseResult {
   def success[A](a: A): ParseResult[A] =
     Either.right(a)
 
+  //Todo: Possibly deprecate???
```

Comment:
I think it's a legit construction from unsafe code.  It's not necessary, but I guess preserves the aliasing of `Either` to a `ParseResult`.  I'm inclined to keep this one.